libs/designsystem/src/lib/components/card/card-header/card-header.component.scss

```diff
@@ -23,3 +36,29 @@ h4 {
   margin: 0;
   font-weight: font-weight('normal');
 }
+
+// TODO: Should 'informational' be added to $notification-colors in general? Would remove the need for mapping to 'semi-light' but could have broader consequences.
```

Comment:
Probably not, as this is the odd case (i.e. toasts can only be success/warning/danger)